src/components/LibraryItem.tsx

```diff
@@ -98,6 +98,12 @@ export class LibraryItem extends React.Component<LibraryItemProps, LibraryItemSt
             iconElement = (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail} />);
         }
 
+        // If it is a section, display the icon (if given) and provide a click interaction.
+        if (this.props.data.itemType === "section") {
+            iconElement = this.props.data.iconUrl ? (<img className={"LibraryItemIcon"} src={this.props.data.iconUrl} onError={this.onImageLoadFail}
```

Comment:
The icon will be replaced with this Dynamo icon: https://github.com/DynamoDS/librarie.js/blob/master/src/resources/icons/Dynamo.svg